src/components/VolumeSize/index.tsx

```diff
@@ -97,37 +109,54 @@ const StyledCursor = styled('div', {
 `
 
 const StyledLabelContainer = styled('div', {
-  shouldForwardProp: prop => !['size', 'type'].includes(prop),
-})`
+  shouldForwardProp: prop => !['size', 'type'].includes(prop.toString()),
+})<StyleProps>`
   display: flex;
   flex-direction: column;
   white-space: nowrap;
   font-weight: 500;
-  line-height: ${({ size }) => (sizes[size] || 1) * 8}px;
+  line-height: ${({ size = 'medium' }) => (sizes[size] || 1) * 8}px;
 
   position: absolute;
   top: 32px;
   ${({ type }) => (type === 'min' ? `left: 0;` : `right: 0;`)}
 `
 
 const StyledLabel = styled('div', {
-  shouldForwardProp: prop => !['type'].includes(prop),
-})`
+  shouldForwardProp: prop => !['type'].includes(prop.toString()),
+})<StyleProps>`
   text-align: ${({ type }) => (type === 'min' ? 'left' : 'right')};
 `
 
-const getPercentUsed = ({ minSize, maxSize, value, isTooBig, isTooSmall }) => {
+const StyledBox = styled(Box)`
+  display: flex;
+  align-items: center;
+`
+
+const getPercentUsed = ({
+  minSize,
+  maxSize,
+  value,
+  isTooBig,
+  isTooSmall,
+}: {
+  minSize: number
+  maxSize?: number
+  value: number
+  isTooBig?: boolean
+  isTooSmall?: boolean
+}): number => {
   const validMinSize = minSize === 0 ? 1 : minSize
   if (!maxSize) {
-    return parseInt((value / validMinSize) * 50, 10)
+    return parseInt(String((value / validMinSize) * 50), 10)
```

Comment:
parseInt is redundant here no ? Both variables are number

Reply:
Yes you're right its actually fun to see ahah